lib/date_time/date_time.c

```diff
@@ -126,11 +126,34 @@ int date_time_now(int64_t *unix_time_ms)
 	return err;
 }
 
+int date_time_now_local(int64_t *local_time_ms)
+{
+	int err;
+
+	if (local_time_ms == NULL) {
+		LOG_ERR("The passed in pointer cannot be NULL");
+		return -EINVAL;
+	}
+	if (!date_time_is_valid()) {
+		LOG_WRN("Valid time not currently available");
+		return -ENODATA;
+	}
+
+	err = date_time_core_now_local(local_time_ms);
+
+	return err;
```

Comment:
Just a suggestion, `err` is not really needed.
```suggestion
	if (local_time_ms == NULL) {
		LOG_ERR("The passed in pointer cannot be NULL");
		return -EINVAL;
	}
	if (!date_time_is_valid()) {
		LOG_WRN("Valid time not currently available");
		return -ENODATA;
	}

	return date_time_core_now_local(local_time_ms);
```